mxcube3/routes/SampleCentring.py

```diff
@@ -11,6 +11,7 @@
 import signals
 import PIL
 import cStringIO
+from . import scutils
```

Comment:
`import scutils` should work exactly the same, no? Since Python first tries to load from current directory.